core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -400,7 +396,7 @@ void updateFee() throws AnchorException {
                     .receiveAmount(null)
                     .senderId(request.getSenderId())
                     .receiverId(request.getReceiverId())
-                    .clientDomain(token.getClientDomain())
+                    .clientId(token.getAccount())
```

Comment:
@JakeUrban, here's where the value was assigned in the platform side.

You'll notice we were indeed sending the clientId and I changed it so we send the sender address instead.

FYI: this value is not being used in the reference server, so you won't find any changes under `anchor-reference-server/*`.